FrontEnd/src/components/NavBar/NavBar.module.scss

```diff
@@ -17,5 +17,5 @@
 }
 
 .logoutText{
-  color: rgba(32, 32, 32, 0.6);
+  color: $background-blur-dark;
 }
```

Comment:
Nitpick: new line at the end. Prettier should make this happen so your prettier isn't working.